lib/tasks/scripts/link_teleconsultation_medical_officers.rb

```diff
@@ -0,0 +1,37 @@
+class LinkTeleconsultationMedicalOfficers
+  class << self
+    def call
+      linked_users_count, unlinked_numbers_count = 0, 0
+
+      Facility.all.each do |facility|
+        facility.teleconsultation_phone_numbers.each do |number|
+          medical_officer = find_medical_officer_by_number(number["phone_number"])
```

Comment:
Finding both linked and unlinked users in the same method is a bit complex 😅 Could we split this up a bit, to something like:

```ruby
phone_number = number["phone_number"]

if already_linked?(phone_number)
  already_linked_users_count += 1
else if linkable?(phone_number)
  link(phone_number)
  linked_users_count += 1
else
  unlinked_users_count += 1
end
```